docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs

```diff
@@ -0,0 +1,121 @@
+                Sentiment_Text = "this film was just brilliant casting location scenery story direction " +
+                "everyone's really suited the part they played and you could just imagine being there robert " +
+                "is an amazing actor and now the same being director  father came from the same scottish " +
+                "island as myself so i loved the fact there was a real connection with this film the witty " +
+                "remarks throughout the film were great it was just brilliant so much that i bought the " +
+                "film as soon as it was released for  and would recommend it to everyone to watch and the " +
+                "fly fishing was amazing really cried at the end it was so sad and you know what they say " +
+                "if you cry at a film it must have been good and this definitely was also  to the two " +
+                "little boy's that played the  of norman and paul they were just brilliant children are " +
+                "often left out of the  list i think because the stars that play them all grown up are " +
+                "such a big profile for the whole film but these children are amazing and should be praised " +
+                "for what they have done don't you think the whole story was so lovely because it was true " +
+                "and was someone's life after all that was shared with us all" } };
+            var dataView = mlContext.Data.ReadFromEnumerable(data);
+
+            // This is the dictionary to convert words into the integer indexes.
+            var lookupMap = mlContext.Data.ReadFromTextFile(Path.Combine(modelLocation, "imdb_word_index.csv"),
+                   columns: new[]
+                   {
+                        new TextLoader.Column("Words", DataKind.TX, 0),
+                        new TextLoader.Column("Ids", DataKind.I4, 1),
+                   },
+                   separatorChar: ','
+               );
+
+            // The model expects the input feature vector to be a fixed length vector.
+            // In this sample, CustomMappingEstimator is used to resize variable length vector to fixed length vector.
+            // The following ML.NET pipeline
+            //      1. tokenzies the string into words, 
+            //      2. maps each word to an integer which is an index in the dictionary ('lookupMap'),
+            //      3. Resizes the integer vector to a fixed length vector using CustomMappingEstimator ('ResizeFeaturesAction')
+            //      4. Passes the data to TensorFlow for scoring.
+            //      5. Retreives the 'Prediction' from TensorFlow and put it into ML.NET Pipeline 
+
+            Action<IMDBSentiment, IntermediateFeatures> ResizeFeaturesAction = (i, j) =>
+            {
+                j.Sentiment_Text = i.Sentiment_Text;
+                var features = i.VariableLenghtFeatures;
+                Array.Resize(ref features, MaxSentenceLenth);
```

Comment:
>Array.Resize(ref features, MaxSentenceLenth); [](start = 16, length = 45)

`Property or Indexer cannot be used as out or ref param`...That's why using features as temporary variable.